lib/sycamore/sycamore/functions/chunker.py

```diff
@@ -1,41 +1,86 @@
 from abc import abstractmethod
-from typing import Any
+from typing import Any, List, TypeVar, Generic
 
+T = TypeVar('T')
 
-class Chunker:
+class Chunker(Generic[T]):
     @abstractmethod
-    def chunk(self, tokens: list[Any]) -> list[Any]:
+    def chunk(self, tokens: List[T]) -> List[List[T]]:
         pass
```

Comment:
Generics were added in python 3.12. We want to be able to support >=3.9,<3.13
https://github.com/aryn-ai/sycamore/blob/96b61e0741b2f763e9341f64d06b45cbdda96928/lib/sycamore/pyproject.toml#L16